src/app/core/admin/admin-menu/admin-menu-item/admin-menu-item.component.html

```diff
@@ -0,0 +1,48 @@
+<h2 mat-dialog-title>
+  {{ !isNew ? 'Edit Menu Item' : 'Adding New Menu Item' }}
+</h2>
+
+<mat-dialog-content class="menu-item-form">
+  <form class="form-grid">
+    <p i18n>
+      Optionally select a record type to automatically use its name, icon and list view (as link) for this menu item.
+      Otherwise, you can select a custom name and icon below.
+    </p>
+
+    <!-- EntityType -->
+    <mat-form-field appearance="fill" class="margin-bottom-large">
+      <mat-label>Entity Type</mat-label>
```

Comment:
can you add `i18n` tags to all labels and buttons here to mark things translatable?